backend/test/akvo/flow_services/end_to_end_test.clj

```diff
@@ -213,7 +213,7 @@
       (is (= ["IN_PROGRESS" "FINISHED_SUCCESS"] (final-report-state-in-flow flow-report-id))))))
 
 (deftest error-in-report-generation
-  (let [survey-id (System/currentTimeMillis)
+  (let [survey-id 43993002
```

Comment:
If I remember correctly, we want to change the survey-id between runs so that we know that each run generates a new report, because flow-services keeps some cache, of which survey-id is part of the key.

If this is the case, maybe the test can copy the pregenerated zip file to the correct place in the s3 mock.

Reply:
Simplest  approach is to begin this test by requesting services to invalidate the cache.

Otherwise, to be a correct published form, surveyId is both supposed to be
1. _in_ the xml file
2. the name of the xml file in the zip archive.
But the report generator never actually checks 1. so some renaming would be enough.